arrows/serialize/protobuf/tests/test_serialize_metadata.cxx

```diff
@@ -39,47 +39,32 @@ TEST( serialize_metadata, metadata )
   // duplicate that collection and make some modifications
   // put both collections in a vector
 
-  static kwiver::vital::metadata_traits traits;
   auto meta_sptr = std::make_shared< kwiver::vital::metadata>();
+  meta_sptr->add< kwiver::vital::VITAL_META_METADATA_ORIGIN >( "test-source" );
+  meta_sptr->add< kwiver::vital::VITAL_META_UNIX_TIMESTAMP >( 12345678 );
+  meta_sptr->add< kwiver::vital::VITAL_META_SENSOR_VERTICAL_FOV >( 12345.678 );
 
   {
-    const auto& info = traits.find( kwiver::vital::VITAL_META_METADATA_ORIGIN );
-    auto item = info.create_metadata_item( kwiver::vital::any(std::string ("test-source")) );
-    meta_sptr->add( std::move( item ) );
+    kwiver::vital::geo_point::geo_2d_point_t geo_2d{ 42.50, 73.54 };
+    kwiver::vital::geo_point pt{ geo_2d, kwiver::vital::SRID::lat_lon_WGS84 };
+    meta_sptr->add< kwiver::vital::VITAL_META_FRAME_CENTER >( pt );
   }
 
   {
-    const auto& info = traits.find( kwiver::vital::VITAL_META_UNIX_TIMESTAMP );
-    auto item = info.create_metadata_item( kwiver::vital::any((uint64_t)12345678) );
-    meta_sptr->add( std::move( item ) );
+    kwiver::vital::geo_point::geo_3d_point_t geo{ 42.50, 73.54, 16.33 };
```

Comment:
The change here seems to replace a TIMESTAMP test with a geographical point test.

Reply:
This is where git's line-by-line changes are a bit deceptive; the actual replacement for `TIMESTAMP` is at line 44. This is replacing lines 64-69.